src/video/cocoa/SDL_cocoamodes.m

```diff
@@ -262,7 +262,8 @@ static SDL_bool GetDisplayMode(_THIS, CGDisplayModeRef vidmode, SDL_bool vidmode
     mode->format = format;
     mode->w = width;
     mode->h = height;
-    mode->refresh_rate = refreshrate;
+    mode->refresh_rate_numerator = refreshrate;
```

Comment:
This can be changed to a proper numerator and denominator value

Reply:
updated, but not test please double check